PascalsTriangle.java

```diff
@@ -0,0 +1,48 @@
+/*Time complexity: O(k*k)
+Space complexity: O(k*k)
```

Comment:
The space complexity would be O(1) if you consider that the extra space you're creating is actually the output